### DISCUSSION TITLE
How do releases of this repo work?

### DISCUSSION THREAD

**Discussion by dbolduc** [open-telemetry/opentelemetry-cpp-contrib#310] [General]
- What is the release cadence?
- Who can initiate a release?
- Is the process documented?
- Is there a single version for all of the repo? Is versioning done per-component?
- Is there a single release for all of the repo? Are releases done per-component?

**Comment by lalitb** [comment]
There is no single release for all the components in this repo. Individual components are responsible for doing a source-only release, vcpkg, Conan, deb/rpm packages, or whatever other way they want to release. The individual components need to document this in their specific README.md.

**Reply by dbolduc** [reply]
Thanks.
